src/components/tuTable/tuTable.vue

```diff
@@ -61,10 +61,16 @@
 							minWidth: header.minWidth,
 							maxWidth: header.maxWidth
 						}"
+                        :draggable="`${isDraggable}`"
```

Comment:
Why do we need interpolation here ?  can't it be just `:draggable="isDraggable"` ?